packages/shorebird_cli/lib/src/commands/patch/patcher.dart

```diff
@@ -67,7 +67,7 @@ https://docs.shorebird.dev/status#link-percentage-ios
   Future<void> assertPreconditions();
 
   /// Asserts that the combination arguments passed to the command are valid.
-  Future<void> assertArgsAreValid() async {}
+  Future<void> assertArgsAreValid();
```

Comment:
I'd revert this change. That way, we don't need to provide no-op implementations for the patchers that don't need them.

Reply:
Yeah, but I made that change because otherwise coverage would complain about the default (empty) implementation not being covered. Happy to revert it though if there is a better approach